web/src/components/storage/dasd/index.js

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright (c) [2024] SUSE LLC
+ *
+ * All Rights Reserved.
+ *
+ * This program is free software; you can redistribute it and/or modify it
+ * under the terms of version 2 of the GNU General Public License as published
+ * by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
+ * more details.
+ *
+ * You should have received a copy of the GNU General Public License along
+ * with this program; if not, contact SUSE LLC.
+ *
+ * To contact SUSE LLC about this file by physical or electronic mail, you may
+ * find current contact information at www.suse.com.
+ */
+
+export { default as DASDPage } from "./DASDPage";
+export { default as DASDTable } from "./DASDTable";
+export { default as DASDFormatProgress } from "./DASDFormatProgress";
```

Comment:
As said in another component, looks like these components are not used outside its own storage/dasd _scope_. Please, do not export them.